[2025-06-13T23:50:30.498+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:50:26.756120+00:00 [queued]>
[2025-06-13T23:50:30.513+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:50:26.756120+00:00 [queued]>
[2025-06-13T23:50:30.513+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-13T23:50:30.532+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_data> on 2025-06-13 23:50:26.756120+00:00
[2025-06-13T23:50:30.540+0000] {standard_task_runner.py:60} INFO - Started process 663 to run task
[2025-06-13T23:50:30.545+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_projeto', 'extract_data', 'manual__2025-06-13T23:50:26.756120+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpseksnv4o']
[2025-06-13T23:50:30.548+0000] {standard_task_runner.py:88} INFO - Job 66: Subtask extract_data
[2025-06-13T23:50:30.623+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:50:26.756120+00:00 [running]> on host a984848cc8d0
[2025-06-13T23:50:30.729+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kayo' AIRFLOW_CTX_DAG_ID='etl_projeto' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T23:50:26.756120+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T23:50:26.756120+00:00'
[2025-06-13T23:51:00.985+0000] {logging_mixin.py:188} INFO - Erro de conex√£o: HTTPSConnectionPool(host='teste-engenheiro.onrender.com', port=443): Read timed out. (read timeout=30). Tentativa 1/5
[2025-06-13T23:51:05.340+0000] {logging_mixin.py:188} INFO - Arquivo salvo em /opt/airflow/data/dados_api.csv
[2025-06-13T23:51:05.343+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-13T23:51:05.361+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_projeto, task_id=extract_data, execution_date=20250613T235026, start_date=20250613T235030, end_date=20250613T235105
[2025-06-13T23:51:05.402+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-13T23:51:05.454+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
